bin/manual-testing/wrapped-message-tests.js

```diff
@@ -0,0 +1,114 @@
+"use strict";
+
+var config, Plugin;
+
+config = require("./manual-testing-config");
+Plugin = require("../../lib/index")(config);
+
+
+/**
+ * Instantiates a plugin and uses its connection to clear all records from the
+ * database.
+ *
+ * @param {ddqPlugin~instance} instance
+ * @param {Function} cb
+ */
+function cleanup(instance, cb) {
+    instance.connection.query("DELETE FROM ??;",
+        [instance.config.table],
+        (wipeErr, data) => {
+            if (wipeErr) {
+                console.log("There was a problem wiping the database");
+                throw new Error(wipeErr);
+            }
+
+            console.log("Cleanup was successful");
+            // This should be 0 in the case of remove
+            console.log("Affected Rows", data.affectedRows);
+            cb(instance);
+        }
+    );
+}
+
+
+/**
+ * Disconnects from the database sometime after the timeout.
+ *
+ * @param {ddqPlugin~instance} instance
+ */
+function handleDisconnect(instance) {
+    setTimeout(() => {
+        instance.disconnect((err) => {
+            if (err) {
+                console.log("There was a problem disconnecting");
+                throw new Error(err);
+            } else {
+                console.log("Disconnected successfully");
+            }
+        });
+    }, 2000);
+}
+
+
+/**
+ * Preps and runs individual tests for wrapped message methods.
+ *
+ * @param {Function} fn The function being tested.
+ * @param {string} query The query to prep for the function.
+ */
```

Comment:
Look through the rest of you code for `throws` and make sure you document then in the jsDoc Block.